adserver/migrations/0034_add_geo_placement.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 2.2.13 on 2020-09-30 20:33
+import django.db.models.deletion
+import django_countries.fields
+import django_extensions.db.fields
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('adserver', '0033_add_keyword_placement_impressions'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
```

Comment:
I believe this migration probably needs to be recreated after merging master. I think you took an older version of the placement migration and that's why we're failing the migrations check.